pyproject.toml

```
@@ -2,7 +2,7 @@
 requires = [
   "setuptools",
   "conan<2.0.0",
-  "scikit-build>=0.11.0",
+  "scikit-build==0.17.1",
```

I would make this a version cap but not restrict it to explicitly the only known working version. Something like: 
```suggestion
  "scikit-build<0.17.2",
```